### ISSUE TITLE
Skip transactions during a particular month?

### ISSUE THREAD

**Issue by kianashui** [jakehilborn/debbit#35]
Is there a way to skip transactions during a particular month or edit the number of transactions already made? I already organically completed my transactions to meet my debit requirement this month so I don't need debbit to automate further purchases until next month. My laptop hasn't been restarted this month (and I haven't opened debbit) so it hasn't tried to do these purchases yet--is there a way to stop this from happening just in this month? Either by somehow manually editing the number of purchases that have already been made this month (ideal since this would be helpful in future months where I have some purchases but not all required for my debit requirement) or forcing it not to do transactions until a specific date?

Any ideas would be much appreciated!

**Comment by jakehilborn** [comment]
Yes this is possible, but it's not user friendly to do in debbit's current state. You'll just need to edit the `purchase_count` fields for whichever month of interest inside the state folder. For example, to skip the rest of September, edit the file `state/debbit_2020_09.txt` and change the `purchase_count` numbers to be greater than the `total_purchases` number you have in config.txt. This gets a little more tricky when you haven't run debbit at all for the month, since this file won't already exist. You can simply create the file though using the example below. You'll need to replace `any_card_description_example_merchant` with your values from config.txt. The format is `<card_description>_<merchant_name>`. The code isn't going to do much bounds checking so you need one fake transaction in there too. Here's something to copy paste that has an example showing 2 merchants.

`state/debbit_2020_09.txt`
```
any_card_description_example_merchant:
  purchase_count: 999
  transactions:
  - unix_time: 0
any_card_description_xfinity_bill_pay:
  purchase_count: 999
  transactions:
  - unix_time: 0
```
